integration-cli/docker_cli_links_test.go

```diff
@@ -141,22 +117,15 @@ func (s *DockerSuite) TestLinksHostsFilesInject(c *check.C) {
 	out, _ = dockerCmd(c, "run", "-itd", "--name", "two", "--link", "one:onetwo", "busybox", "top")
 	idTwo := strings.TrimSpace(out)
 
-	c.Assert(waitRun(idTwo), check.IsNil)
+	c.Assert(waitRun(idTwo), checker.IsNil)
 
 	contentOne, err := readContainerFileWithExec(idOne, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentOne))
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("contentOne: %s", string(contentOne)))
 
 	contentTwo, err := readContainerFileWithExec(idTwo, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentTwo))
-	}
-
-	if !strings.Contains(string(contentTwo), "onetwo") {
-		c.Fatal("Host is not present in updated hosts file", string(contentTwo))
-	}
-
+	c.Assert(err, checker.IsNil, check.Commentf("contentTwo: %s", string(contentTwo)))
+	// Host is not present in updated hosts file
+	c.Assert(string(contentTwo), checker.Contains, "onetwo")
```

Comment:
Yeah, comment above and `check.Commentf` can coexists, for example:
code:
```
    // err shouldn't be nil
    // second line
    // third line
    c.Assert(err, check.NotNil, check.Commentf("This is comment"))
```
output:
```
----------------------------------------------------------------------
FAIL: docker_api_test.go:20: DockerSuite.TestApiGetEnabledCors

docker_api_test.go:25:
    // err shouldn't be nil
    // second line
    // third line
    c.Assert(err, check.NotNil, check.Commentf("This is comment"))
... value = nil
... This is comment


----------------------------------------------------------------------
```
I quite like this format, what do you think? :smile: